packages/webviz-core/src/panels/ThreeDimensionalViz/withTransforms.js

```diff
@@ -44,6 +44,16 @@ function withTransforms<Props: *>(ChildComponent: React.ComponentType<Props>) {
           }
         }
       }
+      const tfs_static = frame[TRANSFORM_STATIC_TOPIC];
+      if (tfs_static) {
+        for (const msg of tfs_static) {
+          for (const tf of msg.message.transforms) {
+            if (tf.child_frame_id !== getGlobalHooks().perPanelHooks().ThreeDimensionalViz.skipTranformFrame) {
```

Comment:
We probably don't need this line, since we only need this for `/tf` for our internal version. Better to keep this complication out of this new code.

Reply:
I see, I'll get rid of it in a moment
Anyways if I look at it correctly there's a typo in the other line  ^^ (skipTranformFrame is missing an 's' I guess) so this should be undefined either way